apps/segment/next.config.js

```diff
@@ -23,6 +23,15 @@ const nextConfig = {
       "@trpc/next",
       "@saleor/apps-shared",
     ],
+    serverComponentsExternalPackages: [
+      "@aws-sdk/client-dynamodb",
+      "@aws-sdk/lib-dynamodb",
+      "@aws-sdk/util-dynamodb",
+      // dependencies of aws-sdk-client-mock
+      "@aws-sdk/client-s3",
+      "@aws-sdk/client-sns",
+      "@aws-sdk/client-sqs",
```

Comment:
question: is this included in the production bundle? that's weird since this is `aws-sdk-client-mock`, right? we shouldn't worry about these packages in the first place, maybe we have some import from test file? or maybe next.js incorrectly bundles devDependencies to production bundle?

Reply:
I've removed this config entry in [`c872a40` (#1699)](https://github.com/saleor/apps/pull/1699/commits/c872a4029eb8daf229c902d0693c2f21fe218c47)